lib/core/src/server/templates/base-manager-head.html

```diff
@@ -27,11 +27,20 @@
   window.onerror = function onerror(message, source, line, column, err) {
     if (window.CONFIG_TYPE !== 'DEVELOPMENT') return;
     // eslint-disable-next-line no-var, vars-on-top
-    var error = { message: err.message, name: err.name, stack: err.stack };
-    window.fetch('/runtime-error', {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ message, source, line, column, error, origin: 'manager' }),
-    });
+    var xhr = new window.XMLHttpRequest();
+    xhr.open('POST', '/runtime-error');
+    xhr.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+    xhr.send(
+      JSON.stringify({
+        /* eslint-disable object-shorthand */
+        message: message,
+        source: source,
+        line: line,
+        column: column,
+        error: { message: err.message, name: err.name, stack: err.stack },
```

Comment:
@ghengeveld I think this is the only change that's required. AFAIK `window.fetch` is available in IE11

Reply:
Not according to MDN or CanIUse...

Reply:
Looks like we have a polyfill in place 👍

Reply:
Okay I tested this and we can't use `fetch`, because the polyfill isn't loaded yet at the time the `onerror` handler can be invoked. This can happen if there's an immediate runtime error in the bundle (e.g. a top-level `throw`). So no, we can't use `fetch`, unfortunately.

Reply:
Ok, I stand corrected! 🙏